src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/utils/AuthException.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.pip.data.management.utils;
+
+@SuppressWarnings("PMD.JUnit5TestShouldBePackagePrivate")
```

Comment:
Can I double check the Suppression here is needed?

Reply:
Removed suppression